location.tex

```diff
@@ -1,3 +1,4 @@
+\todo[inline]{Required sequence issue: \url{https://github.com/SynBioDex/SEPs/issues/98\#issuecomment-603569308}}
```

Comment:
Just place holder for now?

Reply:
whoops; forgot to delete that when I added the sequence change.  I've done so now.